src/components/EntityPage/Participant/ParticipantSummary.js

```diff
@@ -1,32 +1,182 @@
 import * as React from 'react';
 import { compose } from 'recompose';
 import { withTheme } from 'emotion-theming';
-
-import Row from 'uikit/Row';
-import Column from 'uikit/Column';
-
+import VariableSummaryTable from 'uikit/SummaryTable/VariableSummaryTable'
 import { EntityContentSection, EntityContentDivider } from '../';
+import ExternalLink from 'uikit/ExternalLink';
+import { kfWebRoot } from 'common/injectGlobals';
+import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
+import Holder from "./Utils/Holder";
+import styled from "react-emotion";
+import {Div} from "../../../uikit/Core";
+import SequencingDataTable from "./Utils/SequencingDataTable";
+import OtherDataTypesSummaryTable from "./Utils/OtherDataTypesSummaryTable";
+
+//https://kf-qa.netlify.com/participant/PT_CMB6TASJ#summary
 
 const enhance = compose(withTheme);
 
-const ParticipantSummary = () => {
+/**
+ * Sanitizes an array of SummaryTable data.
+ *
+ * Makes boolean values into their corresponding Strings. Transforms null into "Null value".
+ *
+ * @param arr The array
+ * @returns {*} The sannitized array
+ */
+function sannitize(arr) {
+  return arr.map( ele => {
+    let sannitized = ele.summary;
+    if (ele.summary === null) sannitized = "--";
+    else if(typeof ele.summary === "boolean") sannitized = `${sannitized}`;
+
+    return {title: ele.title, summary: sannitized}
+  })
+}
+
+/**
+ * Sometimes, intermediate nested fields are missing.
+ *
+ * Trying to access participant.diagnoses.hits.edges[0].node.diagnosis_category, for example, crashes the page when
+ * node is missing. https://kf-qa.netlify.com/participant/PT_3FV3E420#summary
+ *
+ * Thus, we're wrapping every access in a try catch that simply returns null when the accessor fails.
+ *
+ * We're taking the opportunity to sannitize the return value:
+ *  null = --
+ *  boolean = true/false
+ *
+ * @param obj
+ * @param accessor
+ * @returns {string|*|null|undefined}
+ */
+function getter(obj, accessor) {
```

Comment:
Use `get` and provide a default value (third parameter) which is of the same type as you would get otherwise, so an empty string, empty array, etc...

In cases of Numbers, your return value can depend on what makes more sense to render. Maybe not having a number means `0`, maybe a "no value" indicator e.g. `--` make more sense.